app/routes.py

```diff
@@ -0,0 +1,103 @@
+from flask import Blueprint, json, jsonify, request, make_response
+from app.models.customer import Customer
+from app.models.video import Video
+from app import db
+from flask_sqlalchemy import model
+from sqlalchemy import func
+import requests
+from datetime import timedelta
+
+from tests.test_wave_02 import CUSTOMER_NAME
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+@videos_bp.route("", methods=["GET", "POST"])
+def handle_videos():
+    if request.method == "GET":
+        videos = Video.query.all()
+        videos_response = []
+        for video in videos:
+            videos_response.append(video.create_video_dict())
+        return jsonify(videos_response)
```

Comment:
List comprehension can shorten this block of code. 

```suggestion
        videos = Video.query.all()
        videos_response = [ video.create_video_dict() for video in videos ]
        return jsonify(videos_response)
```